src/test/java/it/pagopa/interop/probing/eservice/registry/reader/unit/config/aws/s3/BucketConfigTest.java

```diff
@@ -0,0 +1,41 @@
+package it.pagopa.interop.probing.eservice.registry.reader.unit.config.aws.s3;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.io.IOException;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.AmazonS3Exception;
+import com.amazonaws.services.s3.model.GetObjectRequest;
+
+import it.pagopa.interop.probing.eservice.registry.reader.config.aws.s3.BucketConfig;
+
+ class BucketConfigTest {
```

Comment:
these tests seem more related to the aws sdk than our code. IMO it is not required unless there is actual logic in the tested methods

Reply:
ok, but i imagine that those classes will be excluded from jacoco coverage, right? Because we made those just to cover with junit most of our application.

Reply:
yes, let's agree on that case by case, but we can exclude classes that are merely wrapping a third party client without adding anything